index.html

```diff
@@ -133,55 +133,74 @@ <h3>Terminology</h3>
-							by a user agent). Note that the author does not have to be human; a machine could
-							automatically produce authored manifests for digital publications.</p>
-					</dd>
+					<p>Depending on the state of a <a>digital publication</a>, its manifest exists in one of two
+						forms:</p>
 
-					<dt>
-						<dfn data-lt="canonical manifest|canonical publication manifest">Canonical Manifest</dfn>
-					</dt>
-					<dd>
-						<p>The Canonical Publication Manifest is a version of the manifest created by user agents when
-							they <a href="#processing-manifest">process the authored manifest</a> and remove all
-							possible ambiguities and incorporate any missing values that can be inferred from another
-							source.</p>
-					</dd>
-				</dl>
+					<dl>
+						<dt>
+							<dfn data-lt="authored publication manifest">Authored Manifest</dfn>
+						</dt>
+						<dd>
+							<p>The Authored Publication Manifest is the serialization of the manifest that the author
+								provides with the digital publication (i.e., prior to be digital publication being
+								processed by a user agent). Note that the author does not have to be human; a machine
+								could automatically produce authored manifests for digital publications.</p>
+						</dd>
 
-				<p>It is possible that an authored manifest is the equivalent of the canonical manifest if there are no
-					ambiguities or missing information, but a canonical manifest only exists after a user agent has
-					inspected the authored manifest as part of the process of obtaining it.</p>
+						<dt>
+							<dfn data-lt="canonical manifest|canonical publication manifest">Canonical Manifest</dfn>
+						</dt>
+						<dd>
+							<p>The Canonical Publication Manifest is a version of the manifest created by user agents
+								when they <a href="#processing-manifest">process the authored manifest</a> and remove
+								all possible ambiguities and incorporate any missing values that can be inferred from
+								another source.</p>
+						</dd>
+					</dl>
+
+					<p>It is possible that an authored manifest is the equivalent of the canonical manifest if there are
+						no ambiguities or missing information, but a canonical manifest only exists after a user agent
+						has inspected the authored manifest as part of the process of obtaining it.</p>
+				</section>
 
-				<p>This part of the specification describes the requirements for creating an authored manifest,
-					regardless of the format of the digital publication. Rules for constructing a canonical manifest
-					from the authored manifest are defined for each specific implementation.</p>
+				<section id="manifest-jsonld">
+					<h4>JSON-LD Authoring and Processing</h4>
 
+					<p>The manifest format enables a number of shorthands to the [[!json-ld]] syntax to simplify
+						authoring manifests. It is not always required that object types be explicitly authored, for
```

Comment:
This is misleading. We do not have "shorthands to the json-ld syntax": we have, in effect, a simple and more complex or, shall we say, a simple and a complete set of information (essentially a Graph), both of them valid JSON-LD, and a transformation that converts the simple graph to the complete one.

(I am not sure how to express this in a consise way.)

Reply:
What about: "The manifest format enables information to be compactly expressed to simplify authoring manifests."

Reply:
Actually... shouldn't we say "The authored manifest enables information to be ..." ? Just to make it clear that this is what the authored manifest is about?

Reply:
Yes, good catch. I've also linked to the definitions of authored and canonical manifest.